app/utils/sanity/start.go

```diff
@@ -78,10 +82,23 @@ var (
 		},
 	}
 
-	checkRangeItems = []rangeItem{{
-		enumRange: []int{int(state.DeliverTxsExecModeSerial), state.DeliverTxsExecModeParallel},
-		name:      state.FlagDeliverTxsExecMode,
-	},
+	checkRangeItems = []rangeItem{
+		{
+			enumRange: []int{int(state.DeliverTxsExecModeSerial), state.DeliverTxsExecModeParallel},
+			name:      state.FlagDeliverTxsExecMode,
+		},
+	}
+
+	conflictPairOpts = []conflictPairWithOption{
+		{
+			conflictPair: conflictPair{
+				configA: boolItem{name: iavl.FlagIavlEnableFastStorage, value: true},
+				configB: funcItem{name: "Upgraded to fast IAVL", expect: false, f: appstatus.IsFastStorageStrategy},
+			},
+			option: boolItem{name: iavl.FlagIavlEnableAutoUpgradeFSS, value: false,
+				hint: fmt.Sprintf("Upgrade to IAVL fast storage may take several hours, if sure set --%s, otherwise unset --%s",
+					iavl.FlagIavlEnableAutoUpgradeFSS, iavl.FlagIavlEnableFastStorage)},
```

Comment:
tips can put it outside in struct conflictPairWithOption

Reply:
done